package.json

```diff
@@ -10,7 +10,8 @@
     "dist"
   ],
   "scripts": {
-    "test": "tsc --noEmit && vitest --run",
```

Comment:
@ariskemper 

Sorry, I didn't notice the change here.

`tsc --noEmit` is necessary to check the types in the test phase.